### ISSUE TITLE
Term with mixed case NOT Captured

### ISSUE THREAD

**Issue by bsebloackai** [geekcomputers/Python#334]
https://github.com/geekcomputers/Python/blob/28b25d4b2a0c484b8e905fd54ec5ffeecf6460c8/calculator.py#L44-L50

If you have term = "Sin(20) + cOs(30)", `term.replace(function, withmath)` on LINE 50 above would not work except the whole term has been converted to lower case. Hence input validation is required.

**Possible Solution:** 
Add `term = term.lower()` prior to the loop on LINE 47. Hence, this will remove the need for having `.lower()` on LINE 48's conditional statement

**Comment by luanpo1234** [comment]
You are trying to replace a substring that does not exist in the string **term**.
So if the string is "Cos(20)", **if function in term.lower():** will find it, but the string won't have changed and term.replace("cos", withmath) will raise an exception.

**Comment by jwadel** [comment]
Yeah. That's the same problem described by @jwadel. Input validation/transformation is required before the loop. Converting the term string to lowercase before looping through the functions list should suffice.
